LDAR_Sim/model_code/methods/sensors/OGI_camera.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+import math
+
+
+def detect_emissions(self, site, leaks_present, equipment_rates, site_true_rate, venting):
```

Comment:
I like this function. I'm wondering if we should have another one for quantify_emissions - not applicable here but would be for screening (when you go from true rate to measured rate). Would be easy to implement. @KeeganShaw-GIS want to make this change?

Reply:
Yeah I like this Idea because its used by most methods, will change ASAP.

Reply:
#82